# Generated by Django 4.2.16 on 2024-09-11 03:05

from django.core import management
from django.db import migrations
from django.db.utils import ProgrammingError


def _rebuild_references_index(apps, schema_editor):
    try:
        management.call_command("rebuild_references_index")
    except ProgrammingError:
        # We get an error running tests when attempting to rebuild the indexes
        # for models registered as Snippets whose migration file runs after this
        # one because the table hasn't been created yet. We just ignore the
        # error.
        pass


class Migration(migrations.Migration):
    dependencies = [
        ("web", "0025_alter_blogpage_body_alter_jobpage_body_and_more"),
    ]

    operations = [
        migrations.RunPython(
            _rebuild_references_index, reverse_code=migrations.RunPython.noop
        )
    ]
